src/components/transactions/Transactions.vue

```diff
@@ -81,15 +87,23 @@ export default {
     accountConnected() {
       return `${this.sharedState.chainId} ${this.sharedState.accountAddress}`
     },
+    tokenTypeSelected() {
+      return this.globalState.currentTokenType
+    },
   },
   watch: {
     accountConnected() {
+      this.transactionsColumns = tokenTypesColumns[this.tokenTypeSelected]
```

Comment:
Sounds like it would be more eficient to have different components for the different tokenTypes instead of creating them dinamically